src/codegeneration/InlineES6ModuleTransformer.js

```diff
@@ -139,6 +138,31 @@ export class InlineES6ModuleTransformer extends ModuleTransformer {
   }
 
   /**
+   * Handles importing modules that have `export * from './path/to/foo.js'` declarations.
+   * @returns {Array} statements
+   */
+  appendExportStatement(statements) {
+    let exportObject = createObjectLiteral(this.getExportProperties());
+    if (this.exportVisitor_.starExports.length) {
+      let starExports = this.exportVisitor_.starExports;
+      let starIdents = starExports.map((moduleSpecifier) => {
+        return createIdentifierExpression(
+          this.getTempVarNameForModuleSpecifier(moduleSpecifier));
+      });
+      statements.push(parseStatement `function exportStar(s, d) { for (let k in s) d[k] = s[k]; }`);
```

Comment:
Why is this function needed? Can it be inlined? Then there is less risk of naming collisions.